Design.md

```diff
@@ -0,0 +1,197 @@
+* Organizer - Secondary user is the organizers/team of the microcosm.
+* Administrator - OSM admin may create microcosms.
+
+# Features
+
+## As a Visitor
+
+### About the map
+
+- [ ] See Notes on the map that need resolution
+
+### About the microcosm
+
+- [x] See a list of microcosms
+- [x] See description of the microcosm
+- [x] See the members of the microcosm
+- [x] See links to facebook and twitter
+- [ ] See a map of the area
+- [ ] See links to members OSM wiki, OSM help, OSM Forum, GitHub, Mapillary, HOT OSM, and Twitter accounts if they exist.
+- [ ] See what the microcosm is working on
+- [ ] See what's new in the microcosm: editing activity, project activity
+- [ ] See feed from twitter
+- [ ] Not be invited more than once per year
+
+### About the microcosm events
+
+- [x] See upcoming events for a microcosm
+- [x] See all upcoming events
+- [ ] See past events
+- [ ] Get directions to teh event
+
+### About the microcosm projects
+
+- [ ] See the projects of the microcosm
+
+## As a Member
+
+### About the microcosm
+
+- [ ] See details about other members
+- [ ] See mayors of neighborhoods
+
+### About the member
+
+- [ ] See their profile
+- [ ] See their upcoming events
+- [ ] See their past events
+- [ ] See their progress
+- [ ] See where they have mapped
+- [ ] Share that they belong to a Local Chapter
+- [ ] Add friends (use OSM profile friends)
+
+### About events
+
+- [ ] Propose a new event
+- [x] RSVP for an event
+
+### About projects
+
+- [ ] Propose a new project
+- [ ] Elect to work on a project
+- [ ] Work on a mapping task (task manager, local project)
+
+### Other
+
+## As an Organizer
+
+### About the mapathon
+
+- [ ] Can adjust the center location and bounds of the AOI
+
+### About the microcosm
+
+- [ ] Manage the description
+- [ ] Set a hashtag for the microcosm
+- [ ] Identify sister microcosms
+
+### Events
+
+- [x] Create an event
+- [ ] Modify an event
+- [ ] Generate Field Papers (Survey Papers)
+
+### Membership
+
+- [ ] Manage members
+- [ ] Send a message to members
+- [ ] Get notified about first time mappers in the area (https://github.com/cliffordsnow/newUsers)
+- [ ] Invite people to join the microcosm
+
+### Quality Assurance
+
+- [ ] Measure the completeness of coverage
+- [ ] Organize feeds (e.g. city bike station locations)
+- [ ] Measure quality assurance
+
+## As an admin
+
+- [ ] Create microcosms
+- [x] Edit microcosms
+- [ ] Periodically scan the wiki for new user groups and local chapters (https://github.com/osmlab/localgroups/blob/master/osmgroups.geojson)
+
+# QA
+
+* https://wiki.openstreetmap.org/wiki/Keep_Right
+* https://www.keepright.at/report_map.php?zoom=12&lat=39.95356&lon=-75.12364
+* https://github.com/keepright/keepright
+* http://osmose.openstreetmap.fr/en/map/
+* https://wiki.openstreetmap.org/wiki/OSM_Inspector
+
+
+# Use Cases
+
+## List reviews in the area
+
+Some people want their changes reviewed.  Provide a list of these for the AOI.
```

Comment:
"API" typo?

Reply:
Area of interest - the area that a microcosm covers.